composer.json

```diff
@@ -7,13 +7,14 @@
   "keywords": ["module", "xp"],
   "require" : {
     "xp-framework/core": "^7.0 | ^6.5",
-    "xp-framework/webservices": "^7.0 | ^6.3",
     "xp-framework/http": "^7.0 | ^6.2",
     "xp-framework/xml": "^7.0 | ^6.0",
-    "xp-framework/scriptlet": "^8.0 | ^6.2",
+    "xp-framework/scriptlet": "^8.0.1 | ^6.2",
```

Comment:
We actually *require* this bugfix to work correctly, so `| ^6.2` is incorrect here.